app/src/main/java/org/openimis/imispolicies/Enquire.java

```diff
@@ -393,8 +393,8 @@ public void run() {
                              TotalDelivieriesLeft = jsonObject.getString("totalDelivieriesLeft").equalsIgnoreCase("null")? "": "TotalDelivieriesLeft: "+jsonObject.getString("totalDelivieriesLeft");
```

Comment:
Fix spell: totalDeliveriesLeft

Reply:
"TotalDelivieriesLeft: " is hardcoded. Please add this in strings.xml.

Reply:
Both tblControls and endpoint use "totalDelivieriesLeft" key, so I kept it there but removed typo from the display. Hardcoded string moved to strings.xml